frontend/src/features/projects/common/tabs/ProjectInformationTab.tsx

```diff
@@ -19,6 +19,7 @@ const ProjectInformationTab: React.FunctionComponent<IProjectInformationTabProps
 
       <UpdateInfoForm
         isReadOnly={isReadOnly}
+        showRisk={true}
```

Comment:
hmm.. what is this?

Reply:
Whether the risk dropdown will be displayed.  As this component is used during submit we don't want the owning agency setting the risk value.  Only SRES should be setting this value.